### ISSUE TITLE
Remove gulp-util dependency (since it's deprecated)

### ISSUE THREAD

**Issue by gverni** [tests-always-included/gulp-css2js#9]
When installing `gulp-css2js` the following warning is shown: 

```
npm WARN deprecated gulp-util@3.0.8: gulp-util is deprecated - replace it, following the guidelines at https://medium.com/gulpjs/gulp-util-ca3b1f9f9ac5
```
Looking at the code, these are the replacement packages needed for this plugin: 

* `gutil.replaceExtension` => The .extname property on Vinyl objects or https://www.npmjs.com/package/replace-ext
* `gutil.PluginError` => https://www.npmjs.com/package/plugin-error

@fidian I can work on a PR if you think you can publish the update after that

**Comment by fidian** [comment]
I don't use gulp any longer, so I would appreciate any help I could get because I won't be able to easily test that the changes work. A PR would be wonderful.

**Comment by fidian** [comment]
Released 1.2.0 with your changes. Thanks for doing all of the work for me.
